### ISSUE TITLE
[Federation] Post edits don't seem to federate to other fediverse platforms

### ISSUE THREAD

**Issue by FynhF** [bookwyrm-social/bookwyrm#2820]
**Describe the bug**
When you edit a post, the changes cannot be seen when viewing the post from within another platform. This means, altough you edited the post, viewers from a different fediverse platform will always see the initial and unedited post (with all its flaws and typos).

I have tested this by viewing my edited Bookwyrm posts from within Mastodon and from within Friendica. In both cases I could not see the changes I made. It also doesn't matter if it is a review, a comment on a book or a quotation.

I have not tested, if the changes federate to other Bookwyrm instances.

**To Reproduce**
1. Publish any post
2. Edit it
3. View the post from a different non-bookwyrm platform like for example Mastodon

**Expected behavior**
When you edit a post, everyone and every platform should be able to see the changes

**Instance**
https://bookrastinating.com (Version 0.5.3)

**Comment by mouse-reeve** [comment]
I believe bookwyrm had editing before mastodon supported it, so we should take a look at how other platforms are now expecting that activities -- it may be a pretty simple fix to get the syntax in line

**Comment by FynhF** [comment]
Thanks for looking over it! :)

I think so, too.
I don't know if this is an important information for the fix, but Mastodon was able to show post changes from other platforms that have post editing before it had this feature itself.

**Comment by chris-y** [comment]
Possibly relevant information:
If you search for the post in Mastodon it does pick up the edited version, and shows the previous edits in Tusky same as it does for Mastodon edited posts.

**Comment by mikelgs1** [comment]
Yes, but I still think this issue can be annoying for users. For example:

- If a book Review is published mentioning some Mastodon user, but the user is misspelled, even though the Bookwyrm review is edited and corrected, the Mastodon user does not receive a notification. The Bookwyrm user has to delete and re-publish the message (that's what happened to me at least).
- Similarly, when a Mastodon user is following and subscribed to a Bookwyrm user, if a review is published only for followers there will be a notification in Mastodon (and consequently it won´t be possible to boost the message). However, even if the Bookwyrm review is edited and set as Public, the Mastodon notification remains the same. Maybe this is Mastodon's or Tusky's "fault" but I thought it interesting to mention.
- ...?

Thank you for Bookwyrm, I love it!!

**Comment by fresseng** [comment]
I confirm this issue is still here. I cannot see the edited post from Mastodon

- Here is the edited Bookwyrm post : https://bw.heraut.eu/user/fresseng/comment/31161 (with a whole paragraph added starting "À ce stade de la lecture, ")
-  Here it is as seen from my Mastodon instance : https://fediscience.org/@fresseng@bw.heraut.eu/114943344871523599 (the paragraph is missing)

This is completely reproducible, for whichever Bookwyrm post : edits are not propagated.

(And by the way, I love Bookwyrm too, this does not prevent me from using it, and it has changed the way I read … A thousand thanks for it !)
